### PR TITLE
Fixing confusing or out-dated records in data base json files

### LINKED ISSUE #255: Need to clean up records in the default EDB json files
Records in the database files for EDB are out-dated or confusing with their naming. Need to create a better default database to be loaded and provide the most up-to-date reaction coefficients and parameter information for each record.

### LINKED ISSUE #259: Clean up / remove out-dated chemistry tests
Some of the chemistry tests should be removed, either because they are superseded by other tests or because they use an outdated approach. 

Tests to remove or update:

- [ ] test_water_softening (update to use smooth min feature used in flowsheet examples) - Wait for issue #258 
- [ ] test_water_softening_Ca (Remove, outdated)
- [ ] test_water_softening_Mg (Remove, outdated)
- [ ] test_phosphorus_removal (Remove, replaced by test_pH_dependent_solubility)

### PR DESCRIPTION
**Pull request by aladshaw3**
## Fixes/Addresses:

#255

## Summary/Motivation:

The default database that gets loaded into MongoDB is a little messy and contains some out-dated information. Updating those records to include most relevant chemistry information. 

## Changes proposed in this PR:
- Updating all .json files for the database
- Renaming some of the 'base' configurations to avoid confusion 

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by dangunter — DISMISSED** [review]
Looks good. I am glad you were able to make the changes to the code as needed, though concerned that the docs may still be insufficient for others to understand how to do the same. But that is an issue for another PR.

File: docs/how_to_guides/edb_cli.rst [hunk 1]
```diff
@@ -21,5 +21,13 @@ stored at ``./docs/schemas/edb_component_schema.html``. Supporting files will al
...
     edb schema -t component -f ./docs/schemas/edb_component_schema.html -o html-js
 
+|arrw| Loading the default database files bundled with WaterTAP. These files get loaded into a
+database named ``electrolytedb``::
 
-.. |arrw| unicode:: U+27A2 .. nice looking arrow glyph
\ No newline at end of file
+    edb load -b
+
+|arrw| Droping the default database named ``electrolytedb``::
```
Comment (adam-a-a) [inline docs/how_to_guides/edb_cli.rst]:
```suggestion
|arrw| Dropping the default database named ``electrolytedb``::
```

File: watertap/edb/data_model.py [hunk 2]
```diff
@@ -862,6 +875,21 @@ def __init__(self, data: Dict, validation=True):
     def reaction_type(self):
         return self.data.get("type", "")
 
+    # # TODO: This function behavior is WRONG!!!
```
Comment (adam-a-a) [inline watertap/edb/data_model.py:878]: @aladshaw3 is this something that should be addressed now or reserved for a later PR?

Comment (aladshaw3) [reply watertap/edb/data_model.py:878]: This is going to be resolved in Dan's next PR

**Review by adam-a-a — APPROVED** [review]
LGTM

**Review by dangunter — APPROVED** [review]
Looks good

**Pull request merged**